Pickers/AbstractActionSheetPicker.m

```diff
@@ -218,7 +254,7 @@ - (void)showActionSheetPicker {
 
 #pragma clang diagnostic push
 #pragma ide diagnostic ignored "UnavailableInDeploymentTarget"
-    if ([UIViewController instancesRespondToSelector:@selector(edgesForExtendedLayout)]) {
```

Comment:
You can test on iOS 7 with tapDismiss action set to cancel. You will find that currently it does not work on iOS 5,6,7. So it's best to remove 'if' check